### DISCUSSION TITLE
Commnet Trouble

### DISCUSSION THREAD

**Discussion by ioorp** [community/community#20219] [Other Feature Feedback, Questions, & Ideas]
Ask ? + Solution

**Comment by radar-blip** [comment]
Yes

**Comment by Herisanto** [comment]
Bagaimana cara mengunakan app open in whatsapp mohon bantuan atau tutorialnya,thanks

**Comment by Herisanto** [comment]
app/src/main/java/io/github/subhamtyagi/openinwhatsapp/fragment/MainFragment.java
